doc/ref/plotting_options/styling.ipynb

```diff
@@ -0,0 +1,519 @@
+   "id": "5fb7798b-4c96-44e7-9822-6da31cd9ceba",
+   "metadata": {},
+   "source": [
+    "# Styling Options\n",
+    "\n",
+    "\n",
+    "```{eval-rst}\n",
+    ".. plotting-options-table:: Styling Options\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1f05c88c-9ba7-4a2c-8f61-24ed4c0fd3cf",
+   "metadata": {},
+   "source": [
+    "(option-bgcolor)=\n",
+    "## `bgcolor`\n",
+    "\n",
+    "The `bgcolor` option sets the background color of the data area of the plot. It accepts any valid CSS color string such as 'white', 'lightgray', or hex codes like '#f0f0f0'. This can be useful to improve contrast or match the theme of a larger dashboard or presentation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5f05fd7d-bf2d-472b-b049-85ade434c653",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.earthquakes(\"pandas\")\n",
+    "\n",
+    "df.hvplot.scatter(\n",
+    "    x='lon',\n",
+    "    y='lat',\n",
+    "    c='mag',\n",
+    "    cmap='inferno_r',\n",
+    "    bgcolor='#f5f5f5',  # light gray background\n",
+    "    title=\"Earthquake Magnitudes by Location\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ebc3e58d-2f6e-4ea5-8958-f21446e9a8a6",
+   "metadata": {},
+   "source": [
+    "(option-color)=\n",
+    "## `color`\n",
+    "\n",
+    "The `color` option sets the color of the plotted elements. It accepts:\n",
+    "- A single color name or hex code (e.g., 'red', '#1f77b4') to apply uniformly.\n",
+    "- A list of colors to cycle through when plotting multiple groups.\n",
+    "- A column name to map colors to values, enabling color encoding.\n",
+    "- An array-like object to assign individual colors.\n",
+    "\n",
+    "If both `color` and `c` are provided, `color` takes precedence. For categorical data, hvPlot automatically uses a discrete colormap unless overridden."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "eb51c665-3b3a-4425-b71a-0723e19cb35f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import hvplot.pandas  # noqa\n",
+    "import hvsampledata\n",
+    "\n",
+    "df = hvsampledata.penguins(\"pandas\")\n",
+    "\n",
+    "# Color by species column\n",
+    "df.hvplot.scatter(\n",
+    "    x=\"flipper_length_mm\",\n",
+    "    y=\"body_mass_g\",\n",
+    "    color=\"species\",\n",
+    "    title=\"Penguins Body Mass by Flipper Length\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7a03fc23-d338-44df-9777-9f06ad1096eb",
+   "metadata": {},
+   "source": [
+    "(option-c)=\n",
+    "## `c`\n",
+    "\n",
+    "Alias for [`color`](option-color) above."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3a1bc8fd-4667-483c-996a-d015da754cff",
+   "metadata": {},
+   "source": [
+    "(option-cmap)=\n",
+    "## `cmap`\n",
```

Comment:
Colorcet should probably be mentioned somewhere in this section. Maybe in a tip admonition to suggest users to use perceptually accurate colormaps.